app/routes/dashboard/apps/$appId/requests.tsx

```diff
@@ -15,7 +15,7 @@ import AppRequestsRateCard, {
 } from "~/components/application/AppRequestsRateCard"
 import { useMatchesRoute } from "~/hooks/useMatchesRoute"
 import { ErrorMetric, getErrorMetrics } from "~/models/errormetrics/errormetrics.server"
-import { getLBOriginClassification } from "~/models/portal.server"
+// import { getLBOriginClassification } from "~/models/portal.server"
```

Comment:
I think that we can remove this.

Reply:
yeah probably, but Ricardo is so close to having a new API for it.